gyp_uv.py

```diff
@@ -34,37 +44,47 @@ def host_arch():
   return machine  # Return as-is and hope for the best.
 
 
-def run_gyp(args):
-  rc = gyp.main(args)
-  if rc != 0:
-    print('Error running GYP')
-    sys.exit(rc)
+def config_vs2017(config_fn):
+  if os.environ.get('VisualStudioVersion') != '15.0': return
+  o = dict()
+  td = o['target_defaults'] = dict()
+  try:
+    td['msbuild_toolset'] = 'v141'
```

Comment:
`os.environ.get('VisualStudioVersion') != '15.0':`  do nothing new (i.e. `return`)
I didn't see any `target_defaults` stuff, so I added it only in the VSver 15.0 case.
See my general comment.